File: twoNumSum/twoNumSum.js
function twoNumberSum(array, targetSum) {
    // Write your code here.
    for (let i = 0; i < array.length - 1; i++) {
        const firstNum = array[i];
        for (let j = i + 1; j < array.length; j++) {
            const secondNum = array[j];
            if (firstNum + secondNum == targetSum) {
                return [firstNum, secondNum];
            }
        }
    }
    return [];
}

var arr = [3, 5, -4, 8, 11, 1, -1, 6]
console.log(twoNumberSum(arr, 10))

File: twoNumSum/twoNumSum.py
def twoNumberSum(array, targetSum):
    for i in range(len(array) - 1):
        firstNum = array[i]
        for j in range(i + 1, len(array)):
            secondNum = array[j]
            if firstNum + secondNum == targetSum:
                return [firstNum, secondNum]
    return []

arr = [3, 5, -4, 8, 11, 1, -1, 6]
print(twoNumberSum(arr, 10))